src/command/process_update_task.rs

```diff
@@ -17,11 +17,15 @@ use frankenstein::Update;
 use frankenstein::UpdateContent;
 use std::fmt::Write;
 use std::str::FromStr;
+use tokio::sync::OnceCell;
 use typed_builder::TypedBuilder;
 
 const BOT_NAME: &str = "@RustWeather77Bot";
 pub const TASK_TYPE: &str = "process_update";
 
+pub static REPO: OnceCell<Repo> = OnceCell::const_new();
+pub static API_CLIENT: OnceCell<ApiClient> = OnceCell::const_new();
```

Comment:
can it be moved inside of ApiClient module as a static function? so both handler and workers can re-use the same API client

Reply:
[Move resources to each file](https://github.com/pxp9/weather_bot_rust/pull/25/commits/5176a3ecf8542c3ab61c769562de985a2a5ef137)